src/local.lambda.ts

```diff
@@ -25,22 +26,27 @@ export class LocalLambda {
   enableCORS: boolean;
   binaryContentTypesOverride: Set<string>;
   pathParamsPattern: string;
+  defaultPath: string;
   app: express.Application;
   requestContext: Record<string, any>;
 
-  constructor(config: LocalLambdaConfig) {
+  constructor(config: LocalLambdaConfig, app?: express.Application, defaultPath?: string) {
     this.handler = config.handler;
     this.port = config.port ?? DefaultPort;
     this.context = config.context || {} as Context;
     this.enableCORS = config.enableCORS ?? true;
     this.binaryContentTypesOverride = new Set(config.binaryContentTypesOverride ?? defaultBinaryContentTypeHeaders);
     this.pathParamsPattern = config.pathParamsPattern ?? DefaultPathParamsPattern;
-    this.app = express();
+    this.app = app || express();
+    this.defaultPath = defaultPath ?? DefaultPathParamsPattern;
     this.requestContext = config.requestContext ?? {};
   }
 
-  run(): void {
-    this.app.all(`${this.pathParamsPattern}*`,async (request: Request, response: Response) => {
+  createServer(): void {
```

Comment:
is this more like create route instead of creating server now?

Reply:
Hey @vcfvct , I've addressed to the PR comments.

Reply:
Hey @vcfvct , I've addressed to the PR comments.